### ISSUE TITLE
ctcp chat to an ipv6 bot causes it to give the default outgoing ipv6 address to connect to

### ISSUE THREAD

**Issue by thommey** [eggheads/eggdrop#90]
| Trac | Data |
| --: | :-- |
| Ticket | 90 |
| Reported by | Robby- |
| Status | new |
| Component | Core |
| Priority | major |
| Milestone | 1.8.0 |
| Keywords | dcc chat ipv6 |
| Version | 1.8.0 CVS |

When you '/CTCP bot CHAT' and that bot is using IPv6 (vhost6) it will give you the machine's default outgoing IPv6 address to connect to instead of the address set with vhost6.

**Comment by vanosg** [comment]
Robby, can you post the config you're using for this?

**Comment by Robby-** [comment]
The entire config? Or any specific settings?

**Comment by Robby-** [comment]
Sorry for the delay.

```
#set vhost4 ""
set vhost6 "my:ipv6:ip::add:res"
#set listen-addr ""
set prefer-ipv6 1
listen +21000 bots
listen +21005 users
listen 21007 users
set nat-ip "my.ipv4.ipadd.res"
```

**Comment by thommey** [comment]
Ok. Not a bug, intended behavior, because listen-addr isn't set. The intention to listen on specific/multiple IPs will be addressed by #151.
